tests/e2e/test_flows_authenticators_webauthn.py

```diff
@@ -95,3 +99,66 @@ def test_webauthn_authenticate_sfe(self):
         login_sfe(self.driver, self.user)
         self.wait_for_url(self.if_user_url("/library"))
         self.assert_user(self.user)
+
+    def register_passkey(self):
```

Comment:
isn't this basically the same as `self.register()`?

Reply:
yup, refactored to simplify